Model/UrlInformation.php

```diff
@@ -248,4 +254,20 @@ public function addAlternateLocale(AlternateLocale $alternateLocale)
 
         return $this;
     }
+
+    /**
+     * @return int
```

Comment:
lets make this `@return int|null The 0-based depth value relative to the base path. Or null if depth is not known.